routers/api/v1/org/action.go

```diff
@@ -384,13 +384,11 @@ func (Action) CreateVariable(ctx *context.APIContext) {
 	//     "$ref": "#/definitions/CreateVariableOption"
 	// responses:
 	//   "201":
-	//     description: response when creating an org-level variable
-	//   "204":
-	//     description: response when creating an org-level variable
+	//     description: response when creating a repo-level variable
```

Comment:
Wait, why `repo-level` when it was org-level above?
```suggestion
	//     description: successfully created the org-level variable
```